### PR TITLE
Add clear data options

### PR DESCRIPTION
**Pull request by valentinb102**
# Reference issue

-   #311 

## Purpose

-   Added 2 options so user can clear just the codes or everything.
-   Reused toggles as checkboxes.

## Approach

-   Combined clear codes function with clear data function using 2 bools. 
-   Is this the right approach? 
-   How should I clear just the settings without clearing codes?

### CODE REVIEW

**Comment by Levminer** [comment]
Thanks for your work so far!

1. The toggles are fine but they should be in a dialog (example: https://github.com/Levminer/authme/blob/dfdf7b0bbd11233814153c57c082473996229f54/interface/windows/import/index.ts#L84 and https://github.com/Levminer/authme/blob/dfdf7b0bbd11233814153c57c082473996229f54/interface/windows/import/import.svelte#L226)

2. You can clear only the codes like this: https://github.com/Levminer/authme/blob/dfdf7b0bbd11233814153c57c082473996229f54/interface/windows/settings/index.ts#L75

**Comment by valentinb102** [comment]
How should I clear just the settings? I guess that would be everything excluding codes and the password?

**Comment by Levminer** [comment]
Yes, you can set settings.settings to 

```ts
{
		language: 0,
		launchOnStartup: true,
		minimizeToTray: true,
		optionalAnalytics: true,
		codesDescription: false,
		blurCodes: false,
		sortCodes: 0,
		codesLayout: 0,	
},
```

**Comment by valentinb102** [comment]
Should the four warning strings be moved into the localization file?

https://github.com/Levminer/authme/blob/a2c1c0986ac1610e935aeedbe9828bc33e6ce528/interface/windows/settings/index.ts#L55
https://github.com/Levminer/authme/blob/a2c1c0986ac1610e935aeedbe9828bc33e6ce528/interface/windows/settings/index.ts#L70
https://github.com/Levminer/authme/blob/a2c1c0986ac1610e935aeedbe9828bc33e6ce528/interface/windows/settings/index.ts#L92
https://github.com/Levminer/authme/blob/a2c1c0986ac1610e935aeedbe9828bc33e6ce528/interface/windows/settings/index.ts#L98

**Comment by Levminer** [comment]
Its' not necessary.

**Comment by valentinb102** [comment]
So is this PR good to go? I think I added everything.

File: interface/windows/settings/index.ts [hunk 1]
```diff
@@ -44,31 +44,90 @@ export const about = async () => {
...
-		return
+		settings.settings.language = 0;
+		settings.settings.launchOnStartup = true;
+		settings.settings.minimizeToTray = true;
+		settings.settings.optionalAnalytics = true;
+		settings.settings.codesDescription = false;
+		settings.settings.blurCodes = false;
+		settings.settings.sortCodes = 0;
+		settings.settings.codesLayout = 0;
+		setSettings(settings)
```
Comment (Levminer) [inline interface/windows/settings/index.ts]: Fix formatting

**Review by Levminer — CHANGES_REQUESTED** [review]
Fix the formatting and good to go! Thanks!

**Review by Levminer — APPROVED** [review]

**Pull request merged**